recipe/sysconfigdata/default/_sysconfigdata_linux.py

```diff
@@ -651,36 +710,35 @@
  'SUBDIRSTOO': 'Include Lib Misc',
  'SYSLIBS': '-lm',
  'SYS_SELECT_WITH_SYS_TIME': 1,
- 'TANH_PRESERVES_ZERO_SIGN': 1,
  'TCLTK_INCLUDES': '-I/opt/anaconda1anaconda2anaconda3/include',
  'TCLTK_LIBS': '-L/opt/anaconda1anaconda2anaconda3/lib '
                '-ltcl@TCLTK_VER@ -ltk@TCLTK_VER@',
  'TESTOPTS': '',
  'TESTPATH': '',
- 'TESTPYTHON': 'LD_LIBRARY_PATH=/opt/anaconda1anaconda2anaconda3/../work/Python-@PYVERFULL@ '
-               './python',
+ 'TESTPYTHON': './python',
  'TESTPYTHONOPTS': '',
- 'TESTRUNNER': 'LD_LIBRARY_PATH=/opt/anaconda1anaconda2anaconda3/../work/Python-@PYVERFULL@ '
-               './python ./Tools/scripts/run_tests.py',
+ 'TESTRUNNER': './python '
+               './Tools/scripts/run_tests.py',
  'TESTTIMEOUT': 1200,
- 'THREADOBJ': 'Python/thread.o',
  'TIMEMODULE_LIB': 'rt',
  'TIME_WITH_SYS_TIME': 1,
  'TM_IN_SYS_TIME': 0,
+ 'TZPATH': '/usr/share/zoneinfo:/usr/lib/zoneinfo:/usr/share/lib/zoneinfo:/etc/zoneinfo:/opt/anaconda1anaconda2anaconda3/share/zoneinfo',
```

Comment:
Editing this is quite cumbersome and error prone. I wonder if there is a better way or if we are stuck with this :-/